camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -1130,15 +1142,15 @@ def cal_fromcorners(self, good):
         if self.camera_model == CAMERA_MODEL.PINHOLE:
             print("stereo pinhole calibration...")
             if LooseVersion(cv2.__version__).version[0] == 2:
-                cv2.stereoCalibrate(opts, lipts, ripts, self.size,
+                ret = cv2.stereoCalibrate(opts, lipts, ripts, self.size,
                                    self.l.intrinsics, self.l.distortion,
```

Comment:
the indentation of the arguments is not adapted after adding "ret = "
same is true for the 2 appearances below.